src/Compilers/CSharp/Test/Symbol/Symbols/ExtendedPartialMethodsTests.cs

```diff
@@ -0,0 +1,1430 @@
+            );
+            comp = CreateCompilation(text1, parseOptions: TestOptions.RegularWithExtendedPartialMethods);
+            comp.VerifyDiagnostics();
+        }
+
+        [Fact]
+        public void Override_WrongAccessiblity()
+        {
+            const string text1 = @"
+partial class D
+{
+    internal override partial string ToString();
+    internal override partial string ToString() => ""hello"";
+}";
+            var comp = CreateCompilation(text1, parseOptions: TestOptions.RegularWithExtendedPartialMethods);
+            comp.VerifyDiagnostics(
+                // (4,38): error CS0507: 'D.ToString()': cannot change access modifiers when overriding 'public' inherited member 'object.ToString()'
+                //     internal override partial string ToString();
+                Diagnostic(ErrorCode.ERR_CantChangeAccessOnOverride, "ToString").WithArguments("D.ToString()", "public", "object.ToString()").WithLocation(4, 38));
+        }
+
+        [Fact]
+        public void Override_AbstractBase()
+        {
+            const string text1 = @"
+abstract class C
+{
+    internal abstract void M1();
+}
+
+partial class D : C
+{
+    internal override partial void M1();
+    internal override partial void M1() { }
+}";
+            var comp = CreateCompilation(text1, parseOptions: TestOptions.RegularWithExtendedPartialMethods);
+            comp.VerifyDiagnostics();
+        }
+
+        [Fact]
+        public void Override_NoAccessibility()
+        {
+            const string text1 = @"
+class C
+{
+    internal virtual void M1() { }
+}
+
+partial class D : C
+{
+    override partial void M1();
+    override partial void M1() { }
+}";
+            var comp = CreateCompilation(text1, parseOptions: TestOptions.RegularWithExtendedPartialMethods);
+            comp.VerifyDiagnostics(
+                // (9,27): error CS8796: Partial method 'D.M1()' must have accessibility modifiers because it has a 'virtual', 'override', 'sealed', or 'new', or 'extern' modifier.
+                //     override partial void M1();
+                Diagnostic(ErrorCode.ERR_PartialMethodWithExtendedModMustHaveAccessMods, "M1").WithArguments("D.M1()").WithLocation(9, 27),
+                // (9,27): error CS0507: 'D.M1()': cannot change access modifiers when overriding 'internal' inherited member 'C.M1()'
+                //     override partial void M1();
+                Diagnostic(ErrorCode.ERR_CantChangeAccessOnOverride, "M1").WithArguments("D.M1()", "internal", "C.M1()").WithLocation(9, 27),
+                // (10,27): error CS8796: Partial method 'D.M1()' must have accessibility modifiers because it has a 'virtual', 'override', 'sealed', or 'new', or 'extern' modifier.
+                //     override partial void M1() { }
+                Diagnostic(ErrorCode.ERR_PartialMethodWithExtendedModMustHaveAccessMods, "M1").WithArguments("D.M1()").WithLocation(10, 27)
+            );
+        }
```

Comment:
>} [](start = 8, length = 1)

Consider asserting value of `comp.GetMember<MethodSymbol>("D.M1").OverriddenMethod`, even if `null`, to catch cases where the behavior changes perhaps unexpectedly.

Reply:
Added some checks of the OverriddenMethod